packages/paste-core/components/menu/stories/index.stories.tsx

```diff
@@ -308,11 +314,44 @@ export const MenuDropdown = (): React.ReactNode => {
           </MediaFigure>
         </MediaObject>
       </MenuItem>
+      <MenuItem variant="destructive" {...menuItemMockProps}>
```

Comment:
question: do I need to run something to update the stories snapshot?

Reply:
`yarn jest -u`